### ISSUE TITLE
Next steps - ready do to migrate out?

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/import-maps#212]
From https://github.com/WICG/import-maps/pull/210#issuecomment-590918055 I got the sense that @domenic feels this incubation is getting kinda stable. Apologies if that's a misinterpretation, and please correct me if I'm wrong. 

What are folks here thinking about next steps for this spec? Should it migrate out to one of the standards orgs?

**Comment by domenic** [comment]
The intended destination for this spec is the HTML Standard. That would require multi-implementation interest, which I haven't seen concrete signs of yet. Is Mozilla interested?

**Comment by marcoscaceres** [comment]
>  Is Mozilla interested?
It's a bit unclear from https://github.com/mozilla/standards-positions/issues/146

**Comment by MylesBorins** [comment]
@domenic with the adoption we are seeing in serverside tools (deno) and build tools (jspm / systemjs) do you still think that the HTML spec is the appropriate place for this proposal to live? I definitely see how that could still make sense, or at least something stand alone in the whatwg that is directly referred to and integrated into HTML.

It seems like what you wanted to see was implementor interest to move this along. I've pinged the mozilla thread again, I've heard Apple might be interested, and can ping at MSFT. How many vendors would you want to hear from to kick off the process of migrating out of WICG?

**Comment by domenic** [comment]
Yes, the HTML spec is still the destination for this.

We need two browsers interested before moving to the HTML Standard, and at least one of them should be planning to ship soon.

**Comment by codehag** [comment]
We don't have strong feelings that external import maps stays in this spec, though it is nice to have. It looks like we can split it out an apply it later, and this would be preferable to upstreaming the whole spec if there currently isn't an implementer with bandwidth to implement. 

So, having the minimal proposal integrated into the HTML Standard would be great. We likely won't start implementation within the next 3 months so you have time.

**Comment by domenic** [comment]
An update: I've sent a HTML spec PR at https://github.com/whatwg/html/pull/8075, with no external import maps support. I suspect this repo will still be useful to have as a coordination point to group together further work, but maybe we'll end up moving everything into a series of HTML issues. I'll certainly remove all the spec text from here, once the HTML spec is merged.

**Comment by domenic** [comment]
I think things are pretty solid! I'm hoping to get full spec review on the pull request from @hiroshige-g, but of course more is always appreciated.

**Comment by domenic** [comment]
We've migrated! https://html.spec.whatwg.org/multipage/webappapis.html#import-maps

https://wicg.github.io/import-maps/ now redirects.

As mentioned above, I still think this repo is a reasonable place for further incubation, but I am also open to closing it all down and migrating everything into HTML Standard issues if the community would prefer.
